pkg/kubelet/cm/libcontainer_manager_mock.go

```diff
@@ -0,0 +1,76 @@
+// +build linux
```

Comment:
I honestly don't get the value these mock objects and their associated tests are providing.

Can't we just have the core CgroupManager interface defined and just have a dumb no-op implementation?

All this mock stuff just looks like noise?  Dont we just want a no-op implementation of CgroupManager interface?  Who careas about libcontainer_ specific piece?

If I am missing something, apologies.